_includes/v20.1/misc/tooling.md

```diff
@@ -0,0 +1,51 @@
+## Support
```

Comment:
@ericharmeling @awoods187 @rafiss Would like your thoughts on this wording.

I've been working on an [internal defintion](https://docs.google.com/document/d/16jDV4BRkTMHNhN1O7NlCVaY4SFDU8WT6juELV3CzTZg/edit?usp=sharing) of full vs beta. In the meantime, this is a proposed wording for this page based on some [research](https://docs.google.com/document/d/16jDV4BRkTMHNhN1O7NlCVaY4SFDU8WT6juELV3CzTZg/edit#heading=h.66ayzme2k264)